integration-tests/actions/actions.go

```diff
@@ -593,12 +579,7 @@ func RewindSimulatedChainToBlockNumber(
 		return 0, fmt.Errorf("error making reorg: %w", err)
 	}
 
-	err = evmClient.WaitForEvents()
-	if err != nil {
-		return 0, fmt.Errorf("error waiting for events: %w", err)
-	}
-
-	latestBlockNumberAfterReorg, err := evmClient.LatestBlockNumber(ctx)
+	latestBlockNumberAfterReorg, err := client.Client.BlockNumber(ctx)
 	if err != nil {
 		return 0, fmt.Errorf("error getting latest block number: %w", err)
 	}
```

Comment:
this method is probably only used by VRF so it's kind of fine to have it migrated to Seth, but:
1. I really wanted to avoid modifying any of existing actions
2. Seth-using actions should live in their own package, now it is mix
3. wonder if CCIP has any plans for docker-based reorg tests, if so they might need this EVMClient-based action

Reply:
@Tofel see https://github.com/smartcontractkit/chainlink/pull/12964#discussion_r1598107462

Reply:
> wonder if CCIP has any plans for docker-based reorg tests, if so they might need this EVMClient-based action

@Tofel if CCIP will need this in the future they can easily add it back. Let's focus on migrating CCIP to Seth instead of creating 2 different functions just in case they may need it at some point while still using the old client.

Reply:
> Seth-using actions should live in their own package, now it is mix

Regarding this, I removed the legacy code so we only have Seth code - 212c2a3fa2bd058d5ad6b8150ceef5ddde3cd8cc